src/main/groovy/org/mockito/release/internal/gradle/ContinuousDeliveryPlugin.java

```diff
@@ -71,28 +70,12 @@ public void execute(Task t) {
         //TODO we should have tasks from the same plugin to have the same group
         //let's have a task maker instance in a plugin that has sets the group accordingly
 
-        //TODO use constants for all task names
-        TaskMaker.execTask(project, "gitAddReleaseNotes", new Action<Exec>() {
```

Comment:
Fantastic change!!! I love when we cut down the complexity, especially when we remove complexity from the fattest plugin!